juicy-html.html

```diff
@@ -36,14 +38,26 @@
             }
         };
         /**
+         * Skips/disregards pending request if any
+         * @return {XMLHttpRequest} skipped XHR request, in fact just `this.pending`
+         */
+        JuicyHTMLPrototype.skip = function() {
```

Comment:
> You prefer them, because they are harder to test and comprehense? For this reason I don't prefer them, sorry :P


Sorry, I meant
> Also from functional, mathematical point of view I prefer functions that returns something. They are _easier_ to test and comprehend.